felt/src/lib.rs

```diff
@@ -960,7 +972,32 @@ mod test {
         fn is_multiple_of_doesnt_panic(ref x in "(0|[1-9][0-9]*)", ref y in "(0|[1-9][0-9]*)") {
             let x = Felt::parse_bytes(x.as_bytes(), 10).unwrap();
             let y = Felt::parse_bytes(y.as_bytes(), 10).unwrap();
-            assert!(x.is_multiple_of(&y));
+            prop_assert!(x.is_multiple_of(&y));
         }
     }
+
+    #[test]
+    // Checks that the result of adding two zeroes falls in range
+    fn sum_zeros_in_range() {
+        let x = &Felt::new(0);
+        let y = &Felt::new(0);
+        let p = &BigUint::parse_bytes(PRIME_STR[2..].as_bytes(), 16).unwrap();
+
+        let result = x + y;
+        let as_uint = &result.to_biguint();
+        assert!(as_uint < p, "{}", as_uint)
+    }
+
+    #[test]
+    // Checks that the result of multiplying two zeroes falls in range
+    fn mul_zeros_in_range() {
+        let x = &Felt::new(0);
+        let y = &Felt::new(0);
+        let p = &BigUint::parse_bytes(PRIME_STR[2..].as_bytes(), 16).unwrap();
+
+        let result = x * y;
+        let as_uint = &result.to_biguint();
+        println!("{}, {}, {}", x, y, result);
+        assert!(as_uint < p, "{}", as_uint)
+    }
```

Comment:
These tests are not proptests, we are using fixed values, so we can compare the results to a fixed value (ie asserting that 0 * 0 = 0) instead of just checking they fall in range

Reply:
Yes, I should have made it clear in the PR's description. In other proptests I did, we decided to also add a unit test to test for the case where the operations are performed on zeros.

What I did is take the existing unit tests and convert them into tests with zeros, and also add proptests. 

I'm waiting for confirmation of whether to use  `FeltBigInt::new(0)` or `FeltBigInt::zero()` to continue with the zero tests. Let me know if you want me to do these in a different PR!

Reply:
Fixed